sources/karmashapes.json

```diff
@@ -0,0 +1,55 @@
+{
+    "description": "Karmashapes is a comprehensive dataset of cities, villages, and habitations in India.", 
+    "edtf:inception": "2023-03-07", 
+    "fullname": "Karmashapes", 
+    "id": 1813460441, 
+    "key": "", 
+    "license": "https://creativecommons.org/licenses/by/4.0/", 
+    "license_text": "You are free to share — copy and redistribute the material in any medium or format, adapt — remix, transform, and build upon the material, for any purpose, even commercially.", 
+    "license_type": "CC-BY", 
+    "name": "karmashapes", 
+    "prefix": "ks", 
+    "remarks": "https://github.com/whosonfirst/whosonfirst-sources/blob/master/sources/karmashapes_remarks.md", 
+    "src:via": [
+        {
+            "context": "India", 
+            "source_link": "https://geonames.nga.mil/geonames/GNSHome/welcome.html", 
+            "source_name": "National Geospatial Intelligence Agency (NGA)", 
+            "source_note": ""
```

Comment:
Is this where we'd indicate the license information? Right now that's only in the remarks file.

Reply:
Might also note which NGA layer, eg "global populated places"?

Reply:
(So never mind, pending seeing draft output of Markdown...)